### PR TITLE
update(docs): adding github icon into readme and docs website

### PR DESCRIPTION
**Pull request by gabriel-cx**
**Proposed Changes**
- adding github icon into readme and docs website

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -76,6 +76,9 @@ KICS scans and detects issues in following Infrastructure as Code solutions:
 <div style="min-width:80;flex:0 0 25%;display:flex;align-items:center;justify-content:center;margin:8px">
         <img alt="Azure Blueprints" src="img/logo-azure-blueprints.png"  width="80" style="min-width:80px">&nbsp;&nbsp;&nbsp;
 </div>
+<div style="min-width:80;flex:0 0 25%;display:flex;align-items:center;justify-content:center;margin:8px">
+        <img alt="Azure Blueprints" src="img/logo-github-icon.png"  width="80" style="min-width:80px">&nbsp;&nbsp;&nbsp;
```
Comment (za) [inline docs/index.md:80]: Hi there. I believe the `alt` text should be `GitHub` something?

Comment (gabriel-cx) [reply docs/index.md:80]: Hi, you are totally right! thank you so much! :D

**Review by cxMiguelSilva — APPROVED** [review]

File: docs/index.md (hunk 1 above)
Comment (za) [reply docs/index.md]: Wow, it's approved and merged now @gabriel-cx Congrats! 🥳

Comment (gabriel-cx) [reply docs/index.md]: Thank you @za for the help!!

**Pull request merged**